demos/action_recognition_demo/python/README.md

```diff
@@ -130,14 +130,15 @@ python3 action_recognition_demo.py \
     -lb <omz_idr>/demos/action_recognition_demo/python/driver_actions.txt
 ```
 
-When a single image is applied as an input, the demo processes and renders it quickly, then exits. To continuously visualize processed results on the screen, apply the `loop` option, which enforces looping over processing a single image.
+>**NOTE**: If you provide a single image as an input, the demo processes and renders it quickly, then exits. To continuously visualize inference results on the screen, apply the `loop` option, which enforces processing a single image in a loop.
+
 You can save processed results to a Motion JPEG AVI file or separate JPEG or PNG files using the `-o` option:
 
 * To save processed results in an AVI file, specify the name of the output file with `avi` extension, for example: `-o output.avi`.
 * To save processed results as images, specify the template name of the output image file with `jpg` or `png` extension, for example: `-o output_%03d.jpg`. The actual file names are constructed from the template at runtime by replacing regular expression `%03d` with the frame number, resulting in the following: `output_000.jpg`, `output_001.jpg`, and so on.
 To avoid disk space overrun in case of continuous input stream, like camera, you can limit the amount of data stored in the output file(s) with the `limit` option. The default value is 1000. To change it, you can apply the `-limit N` option, where `N` is the number of frames to store.
 
->**NOTE**: Windows\* systems may not have the Motion JPEG codec installed by default. If this is the case, OpenCV FFMPEG backend can be downloaded by the PowerShell script provided with the OpenVINO install package and located at `<INSTALL_DIR>/opencv/ffmpeg-download.ps1`. Run the script with Administrative privileges. Alternatively, you can save results as images.
+>**NOTE**: Windows\* systems may not have the Motion JPEG codec installed by default. If this is the case, you can can download OpenCV FFMPEG back end using the PowerShell script, provided with the OpenVINO &trade; install package and located at `<INSTALL_DIR>/opencv/ffmpeg-download.ps1`. The script should be run with administrative privileges if OpenVINO &trade; installed in system protected folder (this is a typical case). Alternatively, you can save results as images.
```

Comment:
- Remove extra 'can': If this is the case, you can can download --> If this is the case, you can download
- Remove coma: using the PowerShell script, provided --> using the PowerShell script provided
- if OpenVINO &trade; installed in system protected folder --> if OpenVINO &trade; is installed in a system protected folder

Reply:
fixed